### PR TITLE
MultiActor Prep - Directory Structure, SignIn/SignUp, Ids, etc.

### PR DESCRIPTION
**Pull request by awatson1978**

### CODE REVIEW

**Comment by rbaertsch** [comment]
what is the purpose of this pull request?

**Comment by awatson1978** [comment]
Hi.  There are a few purposes...  to allow code review between team-members, to confirm that tests run the same on travis as the localhost environment, to isolate changes in separate branches without breaking things, to afford us the merge rollback features if needed, etc.  

As the new member of the team, I'm intentionally avoiding committing changes directly to the master branch (which would invariably break things and/or disrupt other's workflows), and am following [a branching model for larger teams](http://nvie.com/posts/a-successful-git-branching-model/).  When I get something put together that is ready to merge, I'll post a comment to that effect asking for the merge. 

The branch is passing green, and could conceivably be merged right now for a smaller change differential.  All that's in this branch right now is directory restructuring to more clearly identify the different template components in the app (a cosmetic/organizational change to the codebase that doesn't affect the app, but helps me understand what's going on).  

The bigger intent of this branch is to add unique Ids to all the relevant buttons, pages, cards, posts, comments, and other UI elements, so that the test scripts can latch onto them.  In theory, that won't require any structural changes to the underlying code base.  In practice, we'll wind up flushing out lots of bugs as we start putting things under QA.  

I've nearly got the sign-in/sign-up workflows written.  Will definitely get that done today, so I can move forward on the multiactor collaboration script.

**Comment by awatson1978** [comment]
tl;dr - No action needed right now.  I'm staging changes in a way that lets you all review and follow what I'm doing.  I'm moving some of the files into sub-folders so I can get a better sense of all the templates involved in the app.

**Comment by rbaertsch** [comment]
merge done
